### PR TITLE
configure redirect urls as spa instead of web

### PR DESCRIPTION
**Pull request by jeffj6123**
## Purpose
Change the scripts to start using SPA redirects instead of web redirect URIs.
Additionally provide a migration script for existing app registrations

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,19 @@ $Configobj = .\SetupApplications.ps1 -TenantId '<tenant_id>' -ClusterName '<clus
 
 Refer to *CleanupApplications.ps1* and *CleanupUser.ps1* scripts for additional options and examples.
 
+
+### Update an existing Azure Ad Application
+Update an existing Azure Ad Application to migrate the web redirect URIs to SPA redirect URIs.
+This will update the Application registration by moving any web redirect URIs that contain the port(by default 19080) to SPA redirect URIs and ensuring they end with /Explorer/index.html
+The webApplicationId can be found in azure portal by checking the Application (client) ID section of the app registration essentials or the ClusterApplication section of your cluster's  azureActiveryDirectory section of your arm template.
+```PowerShell
+.\UpdateApplication.ps1.ps1 -webApplicationId 86d35e4b-1aa6-4f3c-b0dc-6f007a1f49e8
```
Comment (dragav) [inline README.md:91]: >ps1.ps1 [](http://example.com/codeflow?start=20&length=7)

Typo?

File: UpdateApplication.ps1 [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    $logFile
+)
+
+
+$headers = $null
+. "$PSScriptRoot\Common.ps1"
+$graphAPIFormat = $resourceUrl + "/v1.0/{0}"
+$global:ConfigObj = @{}
+
+function main () {
```
Comment (dragav) [inline UpdateApplication.ps1:24]: >main [](http://example.com/codeflow?start=9&length=4)

I'd suggest a 'whatIf' mode where the diffs are calculated but not applied..

**Review by dragav — APPROVED** [review]
:shipit:

File: UpdateApplication.ps1 (hunk 2 above)
Comment (jeffj6123) [reply UpdateApplication.ps1:30]: Added a whatIf check

**Pull request merged**